resources.yaml

```diff
@@ -297,7 +297,14 @@ deployment:
     group: training-gie-0324
     image: htcondor-secondary
     secondary_htcondor_cluster: true
-
+  training-geno:
+    count: 2
+    flavor: c1.c28m225d50
+    start: 2024-05-07
+    end: 2024-05-10
+    group: training-genome-assembly-2024
+    image: htcondor-secondary
+    secondary_htcondor_cluster: true
```

Comment:
is that still needed?

Reply:
yes until we also get rid of this in vgcn infrastructure

Reply:
yes until we remove it in this repo